sky/execution.py

```diff
@@ -532,9 +532,13 @@ def spot_launch(
             output_prefix=spot.SPOT_CONTROLLER_YAML_PREFIX)
         controller_task = task_lib.Task.from_yaml(yaml_path)
 
-        controller_resources = (controller_resources or
-                                spot.DEFAULT_SPOT_CONTROLLER_RESOURCES)
-        controller_task.set_resources({controller_resources})
+        if controller_resources is not None:
+            print(f'{colorama.Style.DIM}WARNING: Using customized spot '
+                  'controller is experimental and can cause unexpected '
+                  f'hanging or failure.{colorama.Style.RESET_ALL}')
```

Comment:
```suggestion
                  f'hanging or failure for lower hardware specs.{colorama.Style.RESET_ALL}')
```
Or some better wording than that?